packages/cursorless-vscode/src/ide/vscode/VscodeConfiguration.ts

```diff
@@ -42,3 +46,20 @@ export default class VscodeConfiguration implements Configuration {
 
   onDidChangeConfiguration = this.notifier.registerListener;
 }
+
+export function vscodeGetConfigurationString(path: string): string | undefined {
+  const value = vscode.workspace.getConfiguration().get<string>(path);
```

Comment:
This loads the whole configuration, right? Before we just loaded the section we cared about. Probably prematurely optimising by caring about this stuff, but I'm assuming there's a reason VSCode supports retrieving just one section?

Reply:
I'm not really sure why vscode has this section separation. If you like I can split on the last dot and just fetch the section first?

Reply:
Seems reasonable for now

Reply:
done